helper/logging/logging.go

```diff
@@ -18,9 +19,9 @@ const (
 
 var validLevels = []logutils.LogLevel{"TRACE", "DEBUG", "INFO", "WARN", "ERROR"}
 
-// logOutput determines where we should send logs (if anywhere) and the log level.
-func logOutput() (logOutput io.Writer, err error) {
-	logOutput = nil
+// LogOutput determines where we should send logs (if anywhere) and the log level.
+func LogOutput() (logOutput io.Writer, err error) {
+	logOutput = ioutil.Discard
 	envLevel := os.Getenv(EnvLog)
```

Comment:
Travis says `EnvLog` is undefined 

- https://travis-ci.org/hashicorp/terraform/builds/95475472